### PR TITLE
upload images used in .md files

### PR DESCRIPTION
**Pull request by p-zielinski**

### CODE REVIEW

File: scripts/readme-upload-missing-images.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        authorization: "Basic " + btoa(process.env.README_IO_AUTH + ":"),
+      },
+    }
+  );
+  if (response.status !== 200 || !response?.data?.[0]) {
+    throw response;
+  }
+  return response.data[0];
+};
+const getFiles = async (path: string) => {
```
Comment (weronika-kurczyna) [inline scripts/readme-upload-missing-images.ts:84]: it is duplicated in `readme-fix-reference-docs`

File: scripts/readme-upload-missing-images.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import FormData from "form-data";
+
+const readmeUploadMissingImages = async () => {
+  if (process.env.README_IO_AUTH?.length < 10) {
+    console.log("`README_IO_AUTH` was not provided in `.env` file :/");
+    return;
+  }
+  const basePath = path.join(__dirname, "../docs");
+  const pathsToFiles = await getFiles(basePath);
+  for (const pathToFile of pathsToFiles) {
```
Comment (weronika-kurczyna) [inline scripts/readme-upload-missing-images.ts:14]: imo this script is a bit difficult to read.. 
In the future, if we find free time, I would consider separating more functions, e.g. one that processes a file, etc.
maybe create const for `process.env.README_IO_AUTH` etc.
or even adding descriptive comments to make fixing/editing the script easier

File: scripts/readme-upload-missing-images.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import * as fsPromises from "fs/promises";
+import * as fs from "fs";
+import path from "path";
+import axios from "axios";
+import FormData from "form-data";
+
+const readmeUploadMissingImages = async () => {
+  if (process.env.README_IO_AUTH?.length < 10) {
```
Comment (weronika-kurczyna) [inline scripts/readme-upload-missing-images.ts:8]: let's also consider adding more error handling - try - catch

**Review by weronika-kurczyna — APPROVED** [review]
to sum up, the script works, I tested it several times, but in my opinion the code is not easy to read for the first time, more functions describing the process or comments would be useful in the future :)

File: scripts/readme-upload-missing-images.ts (hunk 2 above)
Comment (p-zielinski) [reply scripts/readme-upload-missing-images.ts:14]: added comments, thanks

File: scripts/readme-upload-missing-images.ts (hunk 3 above)
Comment (p-zielinski) [reply scripts/readme-upload-missing-images.ts:8]: why? I don't want to continue on error.

Comment (weronika-kurczyna) [reply scripts/readme-upload-missing-images.ts:9]: ok, I think it's enough
if something goes wrong with this script, I will get this information in the console, right ?

Comment (p-zielinski) [reply scripts/readme-upload-missing-images.ts:9]: yes

**Pull request merged**